FabricTestExample/android/settings.gradle

```diff
@@ -2,3 +2,12 @@ rootProject.name = 'FabricTestExample'
 apply from: file("../node_modules/@react-native-community/cli-platform-android/native_modules.gradle"); applyNativeModulesSettingsGradle(settings)
 include ':app'
 includeBuild('../node_modules/@react-native/gradle-plugin')
+
+includeBuild('../node_modules/react-native') {
+    dependencySubstitution {
+        substitute(module("com.facebook.react:react-android")).using(project(":packages:react-native:ReactAndroid"))
+        substitute(module("com.facebook.react:react-native")).using(project(":packages:react-native:ReactAndroid"))
+        substitute(module("com.facebook.react:hermes-android")).using(project(":packages:react-native:ReactAndroid:hermes-engine"))
+        substitute(module("com.facebook.react:hermes-engine")).using(project(":packages:react-native:ReactAndroid:hermes-engine"))
+    }
+}
```

Comment:
Don't mind this, I'll remove it just before merging, as if there would be any necessity to introduce changes I want to have access to RN code in my Android Studio.

Reply:
Reverted in: https://github.com/software-mansion/react-native-screens/pull/2169/commits/a0d15bbfcda5b84814f217896e667fd516c81577